Reqres/ReqResSwizzle.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface ReqResSwizzle : NSObject
```

Comment:
We would prefer naming which would be consistent with existing code so _ReqresSwizzle_ would be better name